backend/zeno_backend/processing/histogram_processing.py

```diff
@@ -40,16 +40,24 @@ async def histogram_bucket(project_uuid: str, col: ZenoColumn):
 
             if col.data_type == MetadataType.NOMINAL:
                 await db.execute(
-                    sql.SQL("SELECT {} FROM {} GROUP BY {}").format(
+                    sql.SQL(
+                        "SELECT COUNT(*) FROM (SELECT DISTINCT {} FROM {}) AS temp;"
```

Comment:
nit: don't think you need the `AS temp` here, do you?

Reply:
I *think* it speeds it up for some reason, google did some benchmarking.